### PR TITLE
Update the document

### PR DESCRIPTION
**Pull request by jdneo**
@nickzhums You can open this link to have a better view: https://github.com/microsoft/vscode-java-test/blob/b25b7867ce2e964920f01c8aa13eedd135042d98/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,62 +36,38 @@ The [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscja
...
 ### Run/Debug Test Cases
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/run_codelens.png" style="border-radius: 15px" alt="Run Code Lens"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/editor-decoration.png" alt="Run/Debug Test Cases"/>
 </p>
 
-- The extension will generate `Run Test` and `Debug Test` shortcuts (also known as Code Lens) above the class and method definition. Simply click on them will start running or debugging the target test cases.
-
-> Note: If you cannot see the Code Lens in your editor, please refer to this [issue comment](https://github.com/Microsoft/vscode-java-test/issues/470#issuecomment-444681714) as a workaround.
+- The extension will generate shortcuts (the green play button) at the left side of the class and method definition. Simply click on it will start running the target test cases. Or you can right click on it to see more options.
```
Comment (nickzhums) [inline README.md:54]:
```suggestion
- The extension will generate shortcuts (the green play button) on the left side of the class and method definition. To run the target test cases, simply click on the green play button. You can also right click on it to see more options.
```

File: README.md [hunk 2]
```diff
@@ -36,62 +36,38 @@ The [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscja
...
 
 ---
 
 ### Customize Test Configurations
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/configuration.png" style="border-radius: 15px" alt="Customize Test Configurations"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/configuration.png" alt="Customize Test Configurations"/>
 </p>
 
 - Sometimes you may want to customize the configuration for running the test cases. To achieve this, you can add it into your workspace settings under the section: `java.test.config`.
```
Comment (nickzhums) [inline README.md:73]:
```suggestion
- Sometimes you may want to customize the configuration of running the test cases. To achieve this, you can add the configuration into your workspace settings under the section: `java.test.config`.
```

Comment (jdneo) [reply README.md:73]: How about 'you may want to customize the configuration **to run** the test cases'

File: README.md [hunk 3]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/status_bar.png" style="border-radius: 15px" alt="Status Bar"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/test_report.png" alt="View Test Result"/>
 </p>
 
-- After running/debugging the test cases, the status bar will show the final results. Simply click on it to show the Test Report.
-- You can also click the ✔️ or ❌ mark in Code Lens to open the Test Report.
+- After running/debugging the test cases, the state of the related test items will be updated in both editor decoration and test explorer.
+- You can trigger the command `Test: Peek Output` to peek the result view.
+- You can click the links in the stacktrace to navigate to the source location.
```
Comment (nickzhums) [inline README.md:87]:
```suggestion
- You can click the on links in the stack trace to navigate to the source location.
```

File: README.md [hunk 4]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
 <p align="center">
-  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/master/demo/report_navigate.png" style="border-radius: 15px" alt="Status Bar"/>
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/command_palette.png" alt="VS Code Embedded Commands for Testing"/>
 </p>
 
-- You can navigate to the source location of the target test case by clicking the navigate button.
-
-> Note: You can use `java.test.report.showAfterExecution` to configure whether to automatically show the test report after execution. By default, it will be shown when there are failed tests. 
-
+There are some other VS Code embedded commands for testing, which can be found by searching `Test:` in the Command Palette.
```
Comment (nickzhums) [inline README.md:95]:
```suggestion
There are other VS Code embedded commands for testing, which can be found by searching `Test:` in the Command Palette.
```

File: README.md [hunk 5]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
+
+<p align="center">
+  <img src="https://raw.githubusercontent.com/Microsoft/vscode-java-test/cs/update-document/demo/settings.png" alt="VS Code Embedded Settings for Testing"/>
+</p>
+
+There are some other VS Code embedded settings for testing, which can be found by searching `testing` in the Settings view.
+## Project Setup
+### JUnit 5
+
+Please refer to [Getting Started](https://junit.org/junit5/docs/current/user-guide/#overview-getting-started) from the JUnit 5's official document for getting started guide.
```
Comment (nickzhums) [inline README.md:114]:
```suggestion
Please refer to [Getting Started](https://junit.org/junit5/docs/current/user-guide/#overview-getting-started) from the JUnit 5's official document for getting started documentation.
```

File: README.md [hunk 6]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
+
+> Note: If your project does not use build tools(Maven/Gradle/...), please make sure [junit-platform-console-standalone.jar](https://search.maven.org/search?q=g:org.junit.platform%20AND%20a:junit-platform-console-standalone) is on your project classpath.
+
+### JUnit 4
+Please refer to [Download and Install](https://github.com/junit-team/junit4/wiki/Download-and-Install) from the JUnit 4's official document for the getting started guide.
```
Comment (nickzhums) [inline README.md:119]:
```suggestion
Please refer to [Download and Install](https://github.com/junit-team/junit4/wiki/Download-and-Install) from the JUnit 4's official document for the getting started documentation.
```

File: README.md [hunk 7]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
+
+### TestNG
+Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's official document for getting started guide.
```
Comment (nickzhums) [inline README.md:122]:
```suggestion
Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's official document for getting started documentation.
```

File: README.md [hunk 8]
```diff
@@ -100,35 +76,51 @@ Please refer to [TestNG Docs](https://testng.org/doc/) from the TestNG's officia
...
+
 ## FAQ
 If you meet any problem when using the extension, please refer to the [FAQ](https://github.com/microsoft/vscode-java-test/wiki/FAQ) to check if there is an answer to your problem.
```
Comment (nickzhums) [inline README.md:126]: possible to add Github Repo Issues link?

Comment (jdneo) [reply README.md:126]: ok

**Pull request merged**